advanced_samples/CSV/dotnet/CSVtoOCSTest/CSVtoOCSTest.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
```

Comment:
I don't think BlackDuck likes this reference and I don't think we actually use it, I've been removing it.